slakh96/no-mans-land
# Added Preliminary Game Over Screen

- Added Canvas that holds the GameOver Screen
- Modified code that handles Player-Alien collision to display GameOver Screen on collision 
HOW TO TEST: 
- Player prefab has a script called MovePhysics.cs / PlayerInteractions.cs which has an attribute called GOscreen 
- Prefabs folder has a prefab called GOscreen, drag it into the scene and copy it into the GOscreen attribute in the script 
- Now, play the game and die by walking into an Alien (make sure Alien has tag set to "Alien"). Upon collision, game should end and game over screen should be displayed.
